client/src/components/App.jsx

```diff
@@ -13,11 +13,10 @@ class App extends Component {
 
   componentDidMount(){
     const { id } = this.props.match.params;
-    axios.get(`http://localhost:3100/api/listing/${id}`)
+    // axios.get(`http://localhost:3100/api/listing/${id}`) //uncomment when only using this service and not the proxy
+    axios.get(`/api/listing/${id}`)
```

Comment:
Without the localhost part, does this still make the right API call? Whenever I left it like this on my proxy, it would try localhost:3000/api..., which wasn't an actual path.

Reply:
Wes helped me on this one. Because my service is being served up through my proxy, this axios.get request will look to see if my proxy's server is able to handle a GET request, which I have made it be able to do. My proxy will receive this GET request, and then redirect it to localhost:3100, which is where the photo gallery service is running on